### PR TITLE
No issue: Update changelog and version

### PR DESCRIPTION
**Pull request by liuche**
### Pull Request checklist

- [ ] This PR includes thorough **tests** or an explanation of why it does not
- [X] This PR includes a **CHANGELOG entry** or does not need one
- [ ] I have considered adding **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-denied`)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ## [Unreleased]
 ### Added
-- Move Settings into channel on the homescreen (#689)
 
 ### Changed
 
 ### Fixed
 
+
+## [3.8] 2019-04-30
```
Comment (mcomella) [inline CHANGELOG.md]: Should this be the 24th?

File: CHANGELOG.md [hunk 2]
```diff
@@ -220,7 +227,8 @@ The CHANGELOG entries for the releases listed below this were added retroactivel
 ## [1.0] - 2017-12-20
 *Initial release! A browser including home tile shortcuts.*
 
-[Unreleased]: https://github.com/mozilla-mobile/firefox-tv/compare/v3.7-RO...HEAD
+[Unreleased]: https://github.com/mozilla-mobile/firefox-tv/compare/v3.8...HEAD
+[3.7]: https://github.com/mozilla-mobile/firefox-tv/compare/v3.7-RO...v3.8
```
Comment (mcomella) [inline CHANGELOG.md]: The leftmost value should be 3.8, right?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,12 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
 ### Fixed
 
+
+## [3.8] 2019-04-30
+### Added
+- Move Settings into channel on the homescreen (#689)
+
+### Fixed
+- Fix Pocket timeout exceptions (#2095)
```
Comment (mcomella) [inline CHANGELOG.md]: I'm surprised you're adding this and they weren't already added 🤔 : is this a mistake? I can't verify the correctness of adding these.

File: CHANGELOG.md (hunk 1 above)
Comment (liuche) [reply CHANGELOG.md]: We plan to actually release next Tuesday, so generally I push it a few days out.

File: CHANGELOG.md (hunk 3 above)
Comment (liuche) [reply CHANGELOG.md]: I was looking through the commits and didn't see #2095 listed, and it's a fix so I added it in.

**Pull request merged**